resources/serverless/charts/webhook/templates/cluster-role.yaml

```diff
@@ -22,6 +22,7 @@ rules:
       - mutatingwebhookconfigurations
       - validatingwebhookconfigurations
     verbs:
+      - create
```

Comment:
Do we need `create` if we have stub for webhook configuration?

Reply:
Yes. We have a create-if-doesn't-exist logic that requires it. This is needed in case the resources are deleted, as the controller should maintain them.